translations/translations.json

```diff
@@ -6768,7 +6769,7 @@
       "adjustFilters": "አካባቢዎችን ያጥሩ",
       "onlyCurrentlyOpen": "በአሁኑ ሰአት ክፍት የሆኑ ቦታዎችን ብቻ ያሳዩ",
       "poiCategories": "ምድቦች",
-      "showPois": "{{numberOfPois}} ቦታዎችን ያሳዩ",
+      "showPois": " ቦታዎችን ያሳዩ",
```

Comment:
Please delete all of those translations if you don't know whether they are correct or not. 
```suggestion
```
Exceptions are probably only german and english and perhaps if you can speak other languages.

Reply:
I did’t add the field showPois to each language as it was already there. I just added a placeholder before every instance of showPois. I deleted those changes in this commit.

Reply:
Yes, but you change the german version of it which is the base translation. Therefore all translations in other languages are not correct anymore. If they are not deleted, they won't be fixed.

Reply:
okay, i will just revert the translation file to its version before i made any change to it.

Reply:
I will do a commit and if it's not what you meant, feel free to modify it.\n I thought the data in translations.json and deleting some fields would break the app. Guess that's not the case.

Reply:
It's not, if some fields are not there, it just falls back to german.